### ISSUE TITLE
Errors when building the project

### ISSUE THREAD

**Issue by noncom** [Porem5002/flappy-odin#1]
Hi! I tried to build the project following the instruction, which is just to launch `built.bat`, but I get:

```c
PS C:\work\exp\odin\test-ws\flappy-odin> .\build.bat
Building Debug Build...
C:/work/exp/odin/test-ws/flappy-odin/src/main.odin(10:36) Error: Undeclared name: a
        SCREEN_TINT_ON_PAUSE := rl.Color { a = 100 }
                                           ^
C:/work/exp/odin/test-ws/flappy-odin/src/main.odin(11:32) Error: Undeclared name: r
        ... Color { r = 201, g = 213, b = 233, a = 255 } ...
                    ^
C:/work/exp/odin/test-ws/flappy-odin/src/main.odin(11:41) Error: Undeclared name: g
        ...  = 201, g = 213, b = 233, a = 255 } ...
                    ^
C:/work/exp/odin/test-ws/flappy-odin/src/main.odin(11:50) Error: Undeclared name: b
        ...  = 213, b = 233, a = 255 } ...
                    ^
                    ^
C:/work/exp/odin/test-ws/flappy-odin/src/main.odin(12:29) Error: Undeclared name: r
        UI_TEXT_COLOR := rl.Color { r = 255, g = 180, b = 0, a = 255 }
                                    ^
C:/work/exp/odin/test-ws/flappy-odin/src/main.odin(12:38) Error: Undeclared name: g
        UI_TEXT_COLOR := rl.Color { r = 255, g = 180, b = 0, a = 255 }
                                             ^
                                                      ^
C:/work/exp/odin/test-ws/flappy-odin/src/main.odin(12:54) Error: Undeclared name: a
        UI_TEXT_COLOR := rl.Color { r = 255, g = 180, b = 0, a = 255 }
                                                             ^
C:/work/exp/odin/test-ws/flappy-odin/src/obstacles.odin(18:35) Error: Undeclared name: r
        ... Color { r = 28, g = 27, b = 26, a = 255 } ...
                    ^
C:/work/exp/odin/test-ws/flappy-odin/src/obstacles.odin(18:43) Error: Undeclared name: g
        ... r = 28, g = 27, b = 26, a = 255 } ...
                    ^
C:/work/exp/odin/test-ws/flappy-odin/src/obstacles.odin(18:51) Error: Undeclared name: b
        ... g = 27, b = 26, a = 255 } ...
                    ^
C:/work/exp/odin/test-ws/flappy-odin/src/obstacles.odin(18:59) Error: Undeclared name: a
                    ^
```

Is there something I might be missing?

**Comment by Porem5002** [comment]
This might be related to your Odin version and the way the raylib Color type is defined. Can you tell me your odin version?

**Comment by Porem5002** [comment]
The raylib bindings of odin used to define the Color type as a struct which allowed usage of r,g,b,a fields when initializing a color. But now it is defined as an array, that doesn't support that syntax, causing the error. 

Fixed with commit [375bd2d](https://github.com/Porem5002/flappy-odin/commit/375bd2d)
